src/core/processing/qgsprocessingutils.cpp

```diff
@@ -1068,15 +1068,15 @@ QString QgsProcessingUtils::formatHelpMapAsHtml( const QVariantMap &map, const Q
     }
   }
   if ( !outputs.isEmpty() )
-    s += QObject::tr( "<h2>Outputs</h2>\n" ) + outputs;
+    s += QStringLiteral( "<h2>" ) + QObject::tr( "Outputs" ) + QStringLiteral( "</h2>\n" ) + outputs;
 
   s += QLatin1String( "<br>" );
   if ( !map.value( QStringLiteral( "ALG_CREATOR" ) ).toString().isEmpty() )
-    s += QObject::tr( "<p align=\"right\">Algorithm author: %1</p>" ).arg( getText( QStringLiteral( "ALG_CREATOR" ) ) );
+    s += QStringLiteral( "<p align=\"right\">" ) + QObject::tr( "Algorithm author") + QStringLiteral( ": " ) + getText( QStringLiteral( "ALG_CREATOR" ) ) + QStringLiteral( "</p>" );
```

Comment:
I'm personally never sure about this one -- don't we also need to translate ":" for languages which don't use a ":" character? @rduivenvoorde any insights?

Reply:
Yes we do. For eg in Khmer, : is ៖

Reply:
Thanks @nirvn --  that clears up one long-standing mystery for me!

Reply:
Thank you for the clarification and example @nirvn, done in https://github.com/qgis/QGIS/pull/44935/commits/4a6c2508f8d26876f181e817159c51c889020fe3.

Reply:
> Yes we do. For eg in Khmer, : is ៖

Ghee @nirvn, was'nt aware of that. Thanks!